### PR TITLE
My messages page

### PR DESCRIPTION
**Pull request by laura-gonzalez-z**
> This is pretty much working except for the new message state.

1. Changed avatar large size to the one that we will use in user profiles
2. Created a component for cards and moved some styling from homepage to the _cards.scss
3. Added styling for chat card (still missing the styling for different messages inside a chatroom)
4. Fix the @chatrooms in my_messages so it doesn't get all chatrooms, but just the ones that have my current_user.id in the name and order them by descending created_at messages
5. Create the chat partial and add it in my_messages

Screen Shot 2022-11-28 at 10 24 42 AM

### CODE REVIEW

File: app/controllers/chatrooms_controller.rb [hunk 1]
```diff
@@ -14,7 +14,7 @@ def create
 
   def my_messages
     # TODO
-    @chatrooms = Chatroom.all
+    @chatrooms = Chatroom.where("name LIKE ?", "%_#{current_user.id}_%").includes(:messages).order('messages.created_at DESC')
```
Comment (jon-troupe) [inline app/controllers/chatrooms_controller.rb:17]: I'm good to approve but I'd like to understand what the ? in name LIKE does. Is it a wildcard? or does it specifically grab the current_user that you reference later?

Comment (jon-troupe) [reply app/controllers/chatrooms_controller.rb:17]: Sorry, I just remembered that this is to avoid SQL injections, forgot about that.

Comment (laura-gonzalez-z) [reply app/controllers/chatrooms_controller.rb:17]: Oh yeah! Exactly. To be honest I initially had it without the ?, and then checked the docs and decided to add it here 😆

Comment (laura-gonzalez-z) [reply app/controllers/chatrooms_controller.rb:17]: But it's that: a reference to the current_user.id

**Pull request merged**